### ISSUE TITLE
No result issue on web server (windows)

### ISSUE THREAD

**Issue by son-engr-kr** [IntelligentSensingAndRehabilitation/MonocularBiomechanics#2]
<img width="1891" height="756" alt="Image" src="https://github.com/user-attachments/assets/79401ae9-f50e-4101-83b9-6dc7e2b80489" />

Hi, I encountered two issues:

For one video, I got this warning:

```
UserWarning: A function (load_and_visualize_data) returned too many output values (needed: 2, returned: 3). Ignoring extra values.
Possibly due to a part of body being out of frame.
```

After trimming the video and uploading it via the web interface, no output is shown. I completed both Step 1(Keypoint detection) and Step 2(Biomechanical fitting), and tried all options like reload and load data in the visualization tabs, but nothing worked.

Tested on:
OS: Windows 11
CPU: AMD Ryzen Threadripper 5975WX 32-cores
RAM: 128GB
GPU: NVIDIA T400 4GB (We didn't use MJX - gpu version)

PS. We will try GPU version on demo.ipynb and web soon and share the result.

**Comment by jerela** [comment]
Hi, maybe my answer to #3 can help you.

**Comment by son-engr-kr** [comment]
> Hi, maybe my answer to [#3](https://github.com/IntelligentSensingAndRehabilitation/MonocularBiomechanics/issues/3) can help you.

Thank you!
